linkerd/app/inbound/src/target.rs

```diff
@@ -16,7 +16,7 @@ use tracing::debug;
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub struct TcpAccept {
     pub target_addr: SocketAddr,
```

Comment:
should this also be newtyped?

Reply:
It can't be, currently, because it can be either the local address or the origdst. We'll change this, eventually, to only be the origdst.